githubbot/githubbot/http.go

```diff
@@ -244,16 +246,16 @@ func (h *HTTPSrv) formatMessage(convID chat1.ConvIDStr, event interface{}, repo
 			h.Errorf("error getting pull requests from commit: %s", err)
 		}
 
-		// we have to filter PRs that are not against the repo in question
-		var runPRs []*github.PullRequest
+		// look for PR where the base is the provided repo
+		var runPR *github.PullRequest
 		for _, pr := range pullRequests {
 			if pr.GetBase().GetRepo().GetFullName() == repo {
-				runPRs = append(runPRs, pr)
+				runPR = pr
```

Comment:
add a break after this so we maintain the semantics of the first match and not the last